autosklearn/estimators.py

```diff
@@ -358,6 +359,54 @@ def fit(self, **kwargs):
 
         return self
 
+    def fit_pipeline(
+        self,
+        *args,
+        **kwargs: Dict,
+    ) -> Tuple[Optional[BasePipeline], RunInfo, RunValue]:
+        """ Fits and individual pipeline configuration and returns
+        the result to the user.
+
+        The Estimator constraints are honored, for example the resampling
+        strategy, or memory constraints, unless directly provided to the method.
+        By default, this method supports the same signature as fit(), and any extra
+        arguments are redirected to the TAE evaluation function, which allows for
+        further customization while building a pipeline.
+
+        Parameters
+        ----------
+            X: array-like, shape = (n_samples, n_features)
+                The features used for training
+            y: array-like
+                The labels used for training
+            task: int
+                The type of task, taken from autosklearn.constants
+            is_classification: bool
+                Whether the task is for classification or regression. This affects
+                how the targets are treated
+
+        Returns
+        -------
+            pipeline: Optional[BasePipeline]
+                The fitted pipeline. In case of failure while fitting the pipeline,
+                a None is returned.
+            run_info: RunInFo
+                A named tuple that contains the configuration launched
+            run_value: RunValue
+                A named tuple that contains the result of the run
+        """
+        if self.automl_ is None:
+            self.automl_ = self.build_automl(
+                seed=self.seed,
```

Comment:
Can we maybe simplify the call to `self.build_automl`? Most of the calls pass exactly the same arguments taken from `self`, however, having all these arguments suggests that more is happening.